examples/AeroSharp.Examples/Maps/MapGetByRankExample.cs

```diff
@@ -0,0 +1,47 @@
+using AeroSharp.Connection;
+using AeroSharp.DataAccess;
+using AeroSharp.DataAccess.MapAccess;
+using AeroSharp.Examples.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Aerospike.Client;
+
+namespace AeroSharp.Examples.Maps;
+
+public class MapGetByRankExample : IExample
+{
+    private const string SetName = nameof(MapGetByRankExample);
+
+    private readonly IMap<string, string> _map;
+
+    public MapGetByRankExample(IClientProvider clientProvider)
+    {
+        var dataContext = new DataContext(ExamplesConfiguration.AerospikeNamespace, SetName);
+        var mapConfiguration = new MapConfiguration()
+        {
+            Order = MapOrder.KEY_VALUE_ORDERED
+        };
+        _map = MapBuilder.Configure(clientProvider)
+            .WithDataContext(dataContext)
+            .WithMapConfiguration(mapConfiguration)
+            .Build<string, string>(MapExampleData.RecordKey);
```

Comment:
Very smol comment here (and applies to your `ExecuteAsync` method as well. I personally like to add some vertical air in the code to improve readability. It's completely subjective, and maybe personal preference, but I'd lay this out as:

```csharp
var dataContext = new DataContext(ExamplesConfiguration.AerospikeNamespace, SetName);

var mapConfiguration = new MapConfiguration()
{
   Order = MapOrder.KEY_VALUE_ORDERED
};

_map = MapBuilder.Configure(clientProvider)
   .WithDataContext(dataContext)
   .WithMapConfiguration(mapConfiguration) 
   .Build<string, string>(MapExampleData.RecordKey);
```

I wonder if we can make this easier with some fancy `.editorconfig` rules, but that can be a convo for another day. 😅